compose/compose-file/compose-file-v3.md

```diff
@@ -0,0 +1,2725 @@
+---
+description: Compose file reference
+keywords: fig, composition, compose, docker
+redirect_from:
+- /compose/yml
+- /compose/compose-file-v3/
```

Comment:
This redirect doesn't look correct, because this is the same location as the file is in now